front/app/components/Form/Components/Layouts/CLSurveyPageLayout.tsx

```diff
@@ -323,6 +322,7 @@ const CLSurveyPageLayout = memo(
     return (
       <>
         <Box
+          key={`survey-page-${currentPageIndex}`}
```

Comment:
I would just add a comment above this line about why you've added it.

Reply:
The comment is in the commit message, but yes, I can add a comment in the code as well.